https://issues.redhat.com/browse/ACM-22816 Right size for Virt (#8256)
open-cluster-management/rhacm-docs: release_notes/acm_new.adoc

[#observability-new-features]
== Observability

> * Use right-sizing guides to optimize workloads for Virtualization. To learn more, see link:../observability/obs_right_size_intro.adoc#right-size-intro[Right-sizing recommendation guides (Technology Preview)].

Reviewer: Can we say a little more about this in the What's new?

> * Use right-sizing guides to optimize workloads on a cluster and namespace level. To learn more, see link:../observability/obs_right_size.adoc#optimize-work-right-size[Optimizing workloads by using right-sizing guides].

Author: ```suggestion
* Use right-sizing guides to optimize workloads for virtual machines. To learn more, see link:../observability/obs_right_size.adoc#optimize-work-right-size[Optimizing workloads by using right-sizing guides].
```

> * Use right-sizing guides for Virtualization to optimize workloads and understand resource usage acroos your virtual machines. To learn more, see link:../observability/obs_right_size_intro.adoc#right-size-intro[Right-sizing recommendation guides (Technology Preview)].

Author: ```suggestion
* Use `RightSizingRecommendation` to optimize workloads and understand resource usage across your virtual machines. To learn more, see link:../observability/obs_right_size_intro.adoc#right-size-intro[`RightSizingRecommendation` guides (Technology Preview)].
```
